src/components/TheDetails.vue

```diff
@@ -40,7 +40,9 @@ License along with this library.  If not, see <http://www.gnu.org/licenses/>.
 					>
 					<label :for="'detailsToggleCompleted_' + task.uid" />
 				</span>
-
+				<a class="star reactive" @click="togglePinned(task)">
+					<span :class="[{'disabled': task.calendar.readOnly}, iconPinned]" class="icon" />
+				</a>
```

Comment:
Please insert the icon after the star icon. I know it's confusing but since both the star icon and the pinned icon are `float: right` the pinned icon is at the very right now and we should keep the star icon at the very right.